src/playbook.yaml

```diff
@@ -1,12 +1,49 @@
+# mostly following instructions from
+# https://docs.aws.amazon.com/AmazonECS/latest/developerguide/docker-basics.html
+
 - hosts: all
   become: yes
 
   tasks:
-    - name: install system packages
-      shell: "yes | aptdcon --hide-terminal --install 'build-essential'"
+    - name: Upgrade all packages
+      yum: name=* state=latest
+
+    - name: Install system packages
+      ansible.builtin.yum:
+        name: "{{ item }}"
```

Comment:
why is this in each step: `name: "{{ item }}"`?

Reply:
that's what ansible recommends or maybe I followed a pattern established by some yahoo on the internet?